.travis.yml

```diff
@@ -1,36 +1,34 @@
----
 dist: xenial
-
 language: python
-
 python:
-  - "3.6"
-
+- '3.6'
 services:
-  - docker
-
+- docker
 env:
-  - mysql_root_username=root mysql_root_password='12345'
-
+  global:
+  - mysql_root_username=root
+  - mysql_root_password='12345'
+  - mysql_host=localhost
+  - mysql_db=devopsloft
 before_install:
-  - sudo apt-get install -y libmysqlclient-dev
-
+- sudo apt-get install -y libmysqlclient-dev
+- cp configuration.yml web_s2i/configuration.yml
 addons:
   apt:
     update: true
-
-cache:
-  apt: true`
-
 install:
-  - pip install -r requirements-dev.txt
-  - docker build -t devopsloft/devopsloft docker/.
-
+- pip install -r tests/requirements-test.txt
+- docker build -t devopsloft/mysql db_s2i/.
+- docker build -t devopsloft/devopsloft web_s2i/.
 before_script:
-  - docker run -d -p 80:80 -p 3306:3306 --name web devopsloft/devopsloft
-  - docker exec -it web bash -c 'while ! mysqladmin ping --silent; do sleep 3; done'
-
+- docker network create devopsloft_network
+- docker run -d --network devopsloft_network -p 3306:3306 --name db -e MYSQL_ROOT_PASSWORD=12345 -e MYSQL_DATABASE=devopsloft devopsloft/mysql
+- docker logs db
```

Comment:
what is the purpose of the logs here?

Reply:
for debugging, "docker run" does not have exit status nor show errors within docker

Reply:
how is this debugging helpful? you only get part of the data, if you really want debugging, I suggest collecting the logs in case of a failure in the job

Reply:
It actually helped me to see last error in Travis console.   
How can I see those logs in the Travis framework ?

Reply:
the logs are extremely valuable, no doubt about it.
That being said, the command as it is will only show the logs up until the command was executed, which, may or may not be valuable.
According to the docs, you can use `after_failure` section, in which, you can get the logs if there was a failure, and that would be a far better option

Reply:
will do in one of the next PRs